src/main/java/org/marketdesignresearch/mechlib/core/bidder/valuefunction/transform/ValueTransformation.java

```diff
@@ -0,0 +1,51 @@
+package org.marketdesignresearch.mechlib.core.bidder.valuefunction.transform;
+
+import java.math.BigDecimal;
+
+import org.marketdesignresearch.mechlib.core.bid.bundle.BundleExactValueBid;
+import org.marketdesignresearch.mechlib.core.bidder.valuefunction.ValueFunction;
+
+public interface ValueTransformation {

```

Comment:
I needed to add the "old" Strategy library again, because it is used in some other part of the library. However, I renamed it to ValueTransformation as this is what it actually does, while the new Strategy Interfaces goes way beyond this.

Reply:
I like this, good call